packages/govuk-frontend/src/govuk/components/pagination/_index.scss

```diff
@@ -199,17 +199,17 @@
       border-top: 1px solid $govuk-border-colour;
     }
 
-    // Reset both these elements to their inline default, both to ensure that the focus state
-    // for block mode "shrink wraps" text as expected
+    // Reset both these elements to their inline default, both to ensure that
```

Comment:
That second "both" looks a little odd, like there used to be a second reason in the past.